### DISCUSSION TITLE
Removing Old Code Redundancy

### DISCUSSION THREAD

**Discussion by csdechant** [shannon-lab/zapdos#111] [General / Ideas]
Hi everyone, Zapdos is getting to the point were we have some redundancies with older code. Some are within only Zapdos (such as the Drift Diffusion Action replacing the `DriftDiffusion` kernel), but the bigger one is CRANE replacing most of Zapdos old source term kernels. I am opening this discussion to see if we should remove these old objects (since CRANE is downloaded with Zapdos by default), or does anyone see a need to keep these old objects?

**Comment by ghost** [comment]
I'm generally in favor of simplicity and removing what is not being used directly, but oddly enough sometimes the redundant stuff is helpful. For example, DriftDiffusion was useful because it helped me compared how kernels vs BCs were written, but this may be a case where a development guide may be more helpful than keeping old code around. But this is just my rookie opinion!
